.github/pull_request_template.md

```diff
@@ -11,6 +11,7 @@ Before you ask people to review this PR:
 
 - Tests and linters should be passing: `bundle exec rake`
```

Comment:
No extra full stop on this version? :)

Reply:
Sadly I noticed too late, and then forgot :(